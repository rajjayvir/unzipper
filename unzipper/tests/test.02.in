0x02
0x0a 0x10 0x00
0x78 0x18 0x00
0x00 0x00 0x00 0x00
