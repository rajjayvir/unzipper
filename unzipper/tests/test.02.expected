0x0a : 1 0
0x78 : 1 1
